<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>pqDisplayProxyEditor</class>
 <widget class="QWidget" name="pqDisplayProxyEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Actor Control</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Orientation:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QDoubleSpinBox" name="OriginX" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QDoubleSpinBox" name="OriginY" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Translate:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QDoubleSpinBox" name="TranslateY" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QDoubleSpinBox" name="OrientationZ" >
        <property name="maximum" >
         <double>180</double>
        </property>
        <property name="minimum" >
         <double>-180</double>
        </property>
        <property name="singleStep" >
         <double>5</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QDoubleSpinBox" name="TranslateZ" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QDoubleSpinBox" name="ScaleY" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QDoubleSpinBox" name="ScaleZ" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="TranslateX" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QDoubleSpinBox" name="OrientationY" >
        <property name="maximum" >
         <double>180</double>
        </property>
        <property name="minimum" >
         <double>-180</double>
        </property>
        <property name="singleStep" >
         <double>5</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="ScaleX" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Scale:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="OrientationX" >
        <property name="maximum" >
         <double>180</double>
        </property>
        <property name="minimum" >
         <double>-180</double>
        </property>
        <property name="singleStep" >
         <double>5</double>
        </property>
        <property name="value" >
         <double>0</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Origin:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QDoubleSpinBox" name="OriginZ" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QDoubleSpinBox" name="Opacity" >
        <property name="maximum" >
         <double>1</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Style</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Point size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Representation:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Point Id size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Interpolation:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Line width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="StyleRepresentation" />
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="StyleInterpolation" />
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="StylePointSize" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>1</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QDoubleSpinBox" name="StyleLineWidth" >
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>1</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QDoubleSpinBox" name="StylePointIdSize" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>1</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Color</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="ColorMapScalars" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Map Scalars</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Color by:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="ColorInterpolateColors" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Interpolate Colors</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QPushButton" name="ColorActorColor" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Actor Color...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QComboBox" name="ColorBy" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QPushButton" name="ColorEditColorMap" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Edit Color Map...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="ViewGroup" >
     <property name="title" >
      <string>View</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="0" column="1" >
       <widget class="QPushButton" name="ViewZoomToData" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Zoom To Data</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="ViewScalarBar" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Scalar Bar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="ViewData" >
        <property name="text" >
         <string>Data</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="ViewCubeAxes" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Cube Axes</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="ViewLabelPointIds" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Label Point Ids</string>
        </property>
       </widget>
      </item>
      <item rowspan="3" row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>ViewData</tabstop>
  <tabstop>ViewZoomToData</tabstop>
  <tabstop>ViewScalarBar</tabstop>
  <tabstop>ViewCubeAxes</tabstop>
  <tabstop>ViewLabelPointIds</tabstop>
  <tabstop>ColorBy</tabstop>
  <tabstop>ColorMapScalars</tabstop>
  <tabstop>ColorActorColor</tabstop>
  <tabstop>ColorInterpolateColors</tabstop>
  <tabstop>ColorEditColorMap</tabstop>
  <tabstop>StyleRepresentation</tabstop>
  <tabstop>StyleInterpolation</tabstop>
  <tabstop>StylePointSize</tabstop>
  <tabstop>StyleLineWidth</tabstop>
  <tabstop>StylePointIdSize</tabstop>
  <tabstop>TranslateX</tabstop>
  <tabstop>TranslateY</tabstop>
  <tabstop>TranslateZ</tabstop>
  <tabstop>ScaleX</tabstop>
  <tabstop>ScaleY</tabstop>
  <tabstop>ScaleZ</tabstop>
  <tabstop>OrientationX</tabstop>
  <tabstop>OrientationY</tabstop>
  <tabstop>OrientationZ</tabstop>
  <tabstop>OriginX</tabstop>
  <tabstop>OriginY</tabstop>
  <tabstop>OriginZ</tabstop>
  <tabstop>Opacity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
